website/src/documentation/transforms/python/element-wise/filter.md

```diff
@@ -18,25 +18,168 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Filter 
-<table align="left">
-    <a target="_blank" class="button"
+# Filter
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
+<table>
+  <td>
+    <a class="button" target="_blank"
         href="https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.core.html#apache_beam.transforms.core.Filter">
-      <img src="https://beam.apache.org/images/logos/sdks/python.png" width="20px" height="20px"
-           alt="Pydoc" />
-     Pydoc
+      <img src="https://beam.apache.org/images/logos/sdks/python.png"
+          width="20px" height="20px" alt="Pydoc" />
+      Pydoc
     </a>
+  </td>
 </table>
 <br>
+
 Given a predicate, filter out all elements that don't satisfy that predicate.
 May also be used to filter based on an inequality with a given value based
 on the comparison ordering of the element.
 
 ## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
 
-## Related transforms  
+### Function
+
+`Filter` accepts a function that receives the element as the first argument,
```

Comment:
`as the first argument` -> `as its first argument` ?

Reply:
Ended up rewriting it closer to what Rose suggested.